src/astro/files/type/json.py

```diff
@@ -0,0 +1,28 @@
+import io
+
+import pandas as pd
+
+from astro.constants import FileType as FileTypeConstants
+from astro.files.type.base import FileType
+
+
+class Json(FileType):
```

Comment:
I'm fine with using title capitalization for everything. However, if we want to have GCSLocation, this should be renamed to JSON (JavaScript Object Notation).

My preference is to name everything as title (the first letter is capital, others are lowercase) - just to simplify things - but I'm happy either way, as long as we are consistent. 

Also, in the case of databases and file locations, we had suffixed the classes with what they are. Should we name this JsonFileType?

Reply:
1. Added suffix - FileType
2. Changed classes names to -
   CSVFileType, JSONFileType, NDJSONFileType, ParquetFileType